### DISCUSSION TITLE
Some stutters come back with development version since 2022-05-20

### DISCUSSION THREAD

**Discussion by jcc1953** [flybywiresim/aircraft#7232] [General]
Hello,
Since the development version built 2022-05-20-T08:46:20 (name master: 2ebf1d1e) I notice that some stutters are present again when the aircraft is turning, essentially on the ground when taxiing. These stutters disappear with older version.

**Comment by MaexliBaexli** [comment]
Actually i don´t want to hijack your thread, but since a few weeks i also got notice of microstutters on the FBW.
This microstutter problem is also discussed in the MSFS Forum (https://forums.flightsimulator.com/t/microstutter-fbw/533606).
Perhaps a member of the FBW Team can have a look into it, would be great because in my mind one of the best airplanes for MSFS so far :-) Greets, Markus

**Reply by jcc1953** [reply]
Currently the only stuttering problem I observe would be rather a micro-pause problem that occurs about every sixty seconds, the rest of the time everything is fluid.

**Reply by MaexliBaexli** [reply]
In the avsim forum there is also a topic about the stutters with the FBW (https://www.avsim.com/forums/topic/624863-at-a-loss-concerning-the-fbw-a320dev/), they are working on a fix,

Greets,
Markus

**Reply by jcc1953** [reply]
Very interesting because this is exactly my problem. So let's wait.
Thank.
Jean-Claude.
